src/connection_string.ts

```diff
@@ -1096,10 +1096,20 @@ export const OPTIONS = {
     }
   },
   tlsCAFile: {
-    type: 'string'
+    transform({ name, values: [value] }) {
+      if (typeof value === 'string' && value.length === 0) {
```

Comment:
As-is, the options parsing would permit non-string values for `tlsCAFile` and `tlsCertificateKeyFile`.  Do we have tests for this case? 

I think we should:

- if a non-string value is passed, throw an error
- if a non-empty string is passed, throw an error
- only set the option if it is a string and the string is non-empty